### PR TITLE
libvirt_rng: fix count only for arm

### PR DESCRIPTION
**Pull request by dzhengfy**
- dd count value too small on arm will not make the rate control take effect. Fix dd count only for arm and do not impact for other arches.
- Reorg the rng related files into corresponding directory.

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

File: libvirt/tests/cfg/security/rng/libvirt_rng.cfg [hunk 1]
```diff
@@ -90,7 +90,8 @@
         - rng_rate:
             test_qemu_cmd = "yes"
             test_guest = "yes"
-            dd_throughput = "bs=512 count=100"
+            aarch64:
```
Comment (chunfuwen) [inline libvirt/tests/cfg/security/rng/libvirt_rng.cfg:93]: it shows there are 4 files changes, there maybe something wrong when committing

**Review by chunfuwen — APPROVED** [review]
lgtm

**Comment by chunfuwen** [comment]
@dzhengfy maybe we have one test result available

**Review by xiaodwan — APPROVED** [review]

**Review by cliping — APPROVED** [review]
LGTM

**Pull request merged**